### ISSUE TITLE
waitForService does not work properly

### ISSUE THREAD

**Issue by FinnStutzenstein** [OpenSlides/openslides-manage-service#18]
When using with the datastore, it runs into a worker timeout:
```
datastore-writer_1  | [2021-04-08 09:55:59 +0000] [27] [CRITICAL] WORKER TIMEOUT (pid:29)
datastore-writer_1  | [2021-04-08 09:55:59 +0000] [29] [INFO] Worker exiting (pid: 29)
datastore-writer_1  | [2021-04-08 09:55:59 +0000] [30] [INFO] Booting worker with pid: 30
```

The netcat variant works fine and does not hang up the datastore:
```
while ! nc -z "$DATASTORE_WRITER_HOST" "$DATASTORE_WRITER_PORT" 2> /dev/null; do
    echo "waiting for $DATASTORE_WRITER_HOST:$DATASTORE_WRITER_PORT"
    sleep 1
done
```

**Comment by ostcar** [comment]
I don't accept bash in this repo. If waitForService does not work, we can fix it. There is no need for another language.

**Comment by ostcar** [comment]
I can not reproduce your error.

I started the datastore with `make run-dev-verbose` and the manage service with `go build ./cmd/server/ && DATASTORE_READER_HOST=localhost ./server`. Afterwards, I called `go build ./cmd/manage/ && ./manage config get voting` and got the correct answer. I did not get the log messages you described.

What did you do to get error?

**Comment by FinnStutzenstein** [comment]
I used the prod setup in the current openslides4-dev (merged some changed some minutes ago). If you use the [commented-out lines](https://github.com/FinnStutzenstein/openslides-manage-service/blob/prod/entrypoint-setup#L25-L30) you should be able to reproduce it (also using compose, swarm not needed)

**Comment by ostcar** [comment]
I can reproduce it. But I need you input to solve it.

It works without docker (as explained before). It works in the development environment. It works in the production environment with the datastore-reader-service. It only failes in production with the datastore-writer-service.

Do you have an explanation for it? The waitForService function tries to open a tcp-connection the the datastore-writer-service. This does not succeed. Is there something different how the datastore-writer-service listens for connection in production? Why does the same code work for the reader or the writer in development?

**Comment by ostcar** [comment]
I found the bug. The waitForService function opened a connection and did not close it. It seems, that the datastore-writer does not like this. There is probably only one worker. So if a connection is not closed, no other connection can be opened for 30 seconds. I don't know if this is the correct way. For example, if the backend is killed during a connection to the writer, then the writer-service is blocked for 30 seconds.

But for this issue, the solution was easy. Just close the connection. I already did this in #21. The production build should work again with https://github.com/OpenSlides/OpenSlides/pull/6003
